src/index.js

```diff
@@ -1,3 +1,58 @@
 import './styles/main.scss';
 
-alert('Hello if this worked you are not an idiot');
\ No newline at end of file
+const savedItems = [
+    {
+        completed: false,
+        description: 'Exercise',
+        index: 0,
+    },
+    {
+        completed: false,
+        description: 'Study',
+        index: 1,
+    },
+    {
+        completed: false,
+        description: 'Take a walk',
+        index: 2,
+    }
+];
+
+const displayItems = () => {
+    let items = savedItems;
+    
+    document.querySelector('.container').innerHTML = `
+        <div class="title">
+            <h1>Today's to Do</h1>
+            <button type="button" class="reload"><i class="fa-solid fa-arrows-rotate"></i></button>
+        </div>
+        <form class="todo-form">
+            <input type="text" name="text" id="text-add" placeholder="Add to your list...">
+            <button type="submit" id="submit-btn"><i class="fa-solid fa-right-to-bracket"></i></button>
+        </form>
+        <ul class="list-container">
+        </ul>
+        <button type="button" id="delete-btn">Clear all completed</button>
+    `;
+
```

Comment:
- [ ] Kindly sort the list of items before displaying it as per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md#:~:text=The%20list%20should%20appear%20in%20order%20of%20the%20index%20values%20for%20each%20task.). The list should appear in order of the index values for each task.